syntax = "proto2";

package RoboCupSSL;

option go_package = "github.com/RoboCup-SSL/ssl-vision-client/pkg/vision";

import "ssl_vision_detection.proto";
import "ssl_vision_geometry.proto";

message SSL_WrapperPacket {
  optional SSL_DetectionFrame detection = 1;
  optional SSL_GeometryData geometry = 2;
}
